content/en/blog/2025/ambient-performance/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Istio: the most performant way to achieve network security"
+description: Istio's' ambient mode trailblazes as the highest performance security solution in the ecosystem.
+publishdate: 2025-03-12
+attribution: "John Howard (Solo.io)"
+keywords: [istio,performance,ambient]
+---
+
+Encryption in transit is a baseline requirement for almost all Kubernetes environments today, and forms the foundation of a zero-trust security posture. However, when it comes to security, everyone *wants* more of it, in theory.
+The problem, however, is that security doesn’t come for free – it is always a tradeoff between complexity, user experience, performance, and more.
```

Comment:
```suggestion
Encryption in transit is a baseline requirement for almost all Kubernetes environments today, and forms the foundation of a zero-trust security posture. 

However, the challenge with security is that it doesn’t come without a cost: it often involves a trade-off between complexity, user experience, and performance.
```

I don't think the "wants more of it" angle really makes sense any more - the goal here is to reach the same end state, no matter which solution you pick